### PR TITLE
Add a one-liner script

### PR DESCRIPTION
**Pull request by Gentux**
This script aims to provide a simple way to install and test Nanocloud quickly from any linux distribution anywhere.

### CODE REVIEW

File: one-liner.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        exit 2
+    fi
+}
+
+(
+    cd "${NANOCLOUD_DIR}"
+    echo "$(date "${DATE_FMT}") Creating direcotories"
+    mkdir -p dockerfiles/nginx/conf/certificates
+    mkdir -p installation_dir/scripts
+    mkdir -p installation_dir/scripts
```
Comment (dynamiccast) [inline one-liner.sh]: This directory is created twice

File: one-liner.sh [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        wget --quiet "${URL}" -O -
+    else
+        echo "You need *curl* or *wget* to run this script, exiting"
+        exit 2
+    fi
+}
+
+(
+    cd "${NANOCLOUD_DIR}"
+    echo "$(date "${DATE_FMT}") Creating direcotories"
```
Comment (dynamiccast) [inline one-liner.sh]: Typo alert

File: one-liner.sh [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    mkdir -p installation_dir/scripts
+    mkdir -p installation_dir/scripts
+    echo "$(date "${DATE_FMT}") Downloading artifacts"
+    download "${NANOCLOUD_URL}/nanocloud.sh" > nanocloud.sh &
+    download "${NANOCLOUD_URL}/docker-compose.yml" > docker-compose.yml &
+    download "${NANOCLOUD_URL}/dockerfiles/docker-compose.yml" > dockerfiles/docker-compose.yml &
+    download "${NANOCLOUD_URL}/dockerfiles/nginx/conf/certificates/nginx.crt" > dockerfiles/nginx/conf/certificates/nginx.crt &
+    download "${NANOCLOUD_URL}/dockerfiles/nginx/conf/certificates/nginx.key" > dockerfiles/nginx/conf/certificates/nginx.key &
+    download "${NANOCLOUD_URL}/dockerfiles/nginx/conf/nginx.conf" > dockerfiles/nginx/conf/nginx.conf &
+    download "${NANOCLOUD_URL}/scripts/start.sh" > installation_dir/scripts/start.sh &
```
Comment (dynamiccast) [inline one-liner.sh]: Why isn't the guacamole configuration downloaded as well ?

**Comment by dynamiccast** [comment]
Hi,

Thanks for your contribution. I agree v0.2 needs it's own one liner so users could try out Nanocloud just by using binaries.
However, this implementation is not quite like what I had in mind.

First, should not the script have a different name ? `one-liner.sh` does not really describe what it does. What about `install.sh` ? This way we could server it on our website at this URL : `releases.nanocloud.org/install.sh` which would make more sense than `releases.nanocloud.org/one-liner.sh`.

Also, this script fetches artifacts from the indiana release. Should not the repository target be an option ? I understand indiana is currently the only available release for v0.2 but users may potentially want to try out Nanocloud's beta from binaries instead of the current official release in the long term.

One other comment is about this installation directory. Do we really want the installation to be that volatile ? I mean it's great to know Nanocloud's community can be install in one directory without any other dependency but I don't think users expect it to not be available after a reboot. Should we not use the current directory as an installation directory by default ?

Finally, and this is my biggest issue with this current implementation, Nanocloud's continuous integration system currently doesn't push artifacts downloaded on line 57 to 63 and I don't think it should.
This would force us to upload files from the repository after each build just so they could be eventually downloaded back later from a one-liner installation.
Though I am no sure how this should be handled since the one-liner still needs to perform the installation from itself. I just have a feeling this will create confusion and maintenance issue in the future.

I hope we could iterate on this and that my comments make sense :)

File: installer/entrypoint.sh [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+#
+# Nanocloud community is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+cp -r /community//* /var/lib/nanocloud
```
Comment (Gentux) [inline installer/entrypoint.sh]: There is a _/_ that shouldn't be here

**Comment by Gentux** [comment]
I'm not sure the installer script is usable out-of-the box.

@dynamiccast can you make a comment about this, or udpate README file ?
For README file, I began to rewrite it on my repos, with branch _update_readme_

**Comment by dynamiccast** [comment]
It doesn't work out of the box yet because binaries have not been pushed to the Docker hub yet.
Though the script needs a little modification so it looks for community installer on the Docker Hub.

**Pull request merged**